spec/models/cart_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe Order, type: :model do
+
+  before :each do
+    destination = Destination.create(continent: "South America",
+    description: "South American description")
+    activity = Activity.create(name: "Hiking", description: "Move your legs")
+    @trip = Trip.create(destination_id: destination.id, activity_id: activity.id,
+    name: "Machu Picchu", trip_cost: 5000.00, total_cost: 5000.00 )
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside parentheses detected.